### PR TITLE
Added insertion sort for plain C

### PR DESCRIPTION
**Pull request by JanetLaysEggs**
## Have you read the Contribution Guidelines?

Yes

## Description

Added an implementation of insertion sort in plain C (please see comment in code about the limitations of the language and my workaround).

## Checklist
- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the README.md and link to my code.
 
 
## Related Issues (Mention Issue no. like #xxx)

Fixes Issue 
<br/>
#13

### CODE REVIEW

File: C/InsertionSort.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    int temp = list[i];
+    int j = i-1;
+    for (j; ((temp < list[j]) && (j > -1)); j--) {
+      list[j+1] = list[j];
+    }
+    list[j+1] = temp;
+  }
+  dirtyPrintArray(list,len);
+  return(0);
+}
```
Comment (sukritishah15) [inline C/InsertionSort.c:27]: Below this line (27, end of file), add Sample I/O, Time and Space Complexity in a single multiline comment.

**Review by sukritishah15 — CHANGES_REQUESTED** [review]
@mephi007 @manan025 @mrjoshi0147

**Comment by mephi007** [comment]
hey @JanetLaysEggs, along with @sukritishah15 's comment, can you please resolve merge conflicts too.

**Comment by JanetLaysEggs** [comment]
> hey @JanetLaysEggs, along with @sukritishah15 's comment, can you please resolve merge conflicts too.

Sure will do in the morning 😊.

**Comment by sukritishah15** [comment]
@JanetLaysEggs  - Please make the required changes in 3-4 hours.
As we will re-review this PR in 3-4 hours.

Else, we will have to close this PR due to management purposes.

Thank you!

**Comment by sukritishah15** [comment]
You can create a new PR, but, we are closing this for now.
Thank you!

**Pull request closed without merging**